manual/mapper/config/README.md

```diff
@@ -31,6 +31,8 @@ configuration (make sure you use version 3.5 or higher):
       <artifactId>maven-compiler-plugin</artifactId>
       <version>3.8.1</version>
       <configuration>
+        <source>1.8</source> <!-- (or higher) -->
+        <target>1.8</target> <!-- (or higher) -->
```

Comment:
Not directly related, but I noticed the build fails without this, better be copy-paste safe.